File: system-test/dotnet/SystemTests/Legacy/Mod11/E2eTests/Base/BaseE2eTest.cs
using Dsl.Port;
using Dsl.Core.Shared;
using SystemTests.Legacy.Mod11.Base;

namespace SystemTests.Legacy.Mod11.E2eTests.Base;

public abstract class BaseE2eTest : BaseScenarioDslTest
{
    protected override ExternalSystemMode? GetFixedExternalSystemMode()
    {
        return ExternalSystemMode.Real;
    }
}

File: system-test/java/src/test/java/com/mycompany/myshop/systemtest/legacy/mod11/e2e/base/BaseE2eTest.java
package com.mycompany.myshop.systemtest.legacy.mod11.e2e.base;

import com.mycompany.myshop.systemtest.legacy.mod11.base.BaseScenarioDslTest;
import com.mycompany.myshop.systemtest.configuration.ExternalSystemMode;

public abstract class BaseE2eTest extends BaseScenarioDslTest {
    @Override
    protected ExternalSystemMode getFixedExternalSystemMode() {
        return ExternalSystemMode.REAL;
    }
}




